docs/admin/node.md

```diff
@@ -258,6 +258,29 @@ Set the `cpu` and `memory` values to the amount of resources you want to reserve
 Place the file in the manifest directory (`--config=DIR` flag of kubelet).  Do this
 on each kubelet where you want to reserve resources.
 
+### Horizontal auto-scaling of nodes
+
+If you are using GCE, you can configure your cluster so that the number of nodes will be automatically scaled based on their utilization.
```

Comment:
sorry, didn't realize it was also memory -- so s/utilization/CPU and memory utilization/